addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -567,6 +568,13 @@ export class HtmlField extends Component {
             ids: attachment
         }]));
     }
+    _onDblClickEditableMedia(ev) {
+        const el = ev.target;
+        if (el.nodeName === 'IMG' && el.src) {
+            this.wysiwyg.showImageFullscreen(el.src);
+            return true;
```

Comment:
Why `return true` ?

Reply:
You know what that's an excellent question. Maybe I had a stroke?

Reply:
I see in `wysiwyg.js` I do `if (!this.options.onDblClickEditableMedia?.(ev)) {` so that would be why but that's a bad pattern. How about I remove that `return true` and replace the bad pattern with:
```js
if (this.options.onDblClickEditableMedia && ev.target?.nodeName === 'IMG' && ev.target?.src) {
    this.options.onDblClickEditableMedia(ev);
} else {
    this._onDblClickEditableMedia(ev);
}
```
?

Reply:
Yes please !